CONTRIBUTING.md

```diff
@@ -0,0 +1,14 @@
+# Contributing to mctp-rs
+
+If you wish to contribute, please file a pull request with the following
+characteristics.
+
+- Include a `Signed-off-by` trailer at the end of each commit log message to
+  assert the Developer Certificate of Origin (DCO). `git commit -s` makes this
+  easy.
+
+  XXX: link to something, e.g. https://developercertificate.org/?
```

Comment:
Yep, that's the right link. I'd maybe expand on this a little, perhaps something like:

```
-  Include a `Signed-off-by` trailer at the end of each commit log message to
   indicate that you agree to the Developer Certificate of Origin (DCO).
   `git commit -s` makes adding the `Signed-off-by` tag easy.

   In short, this asserts that you have the right to contribute this change - either
   being your own work, or you are submitting it under a suitable license.

   The text of the DCO is available at <https://developercertificate.org/>
```

Reply:
Updated :+1: